src/components/Footer.tsx

```diff
@@ -0,0 +1,28 @@
+import Link from "next/link";
+
+import { Button } from "./ui/button";
+
+const Footer = () => {
+  return (
+    <footer className="w-full bg-slate-900 py-8">
+      <div className="text-white text-center text-sm">
+        <p className="py-4">フロントエンドカンファレンス北海道2024</p>
+        <p className="py-4">
+          @2024 フロントエンドカンファレンス
+          <br />
+          北海道実行委員会
+        </p>
+        <Button asChild>
+          <Link
+            href="https://aback-jasmine-06b.notion.site/cf184497a6414a97aea49c1f1b2f5b5f"
+            className="py-4"
+          >
+            プライバシーポリシー
+          </Link>
+        </Button>
```

Comment:
スミマセン
こっちも送信したつもりが送信できてなかったです

buttonのvariantにlinkを使うのと、文字の太さは周りと合わせたほうが良さそうなので対応ください
```suggestion
        <Button valiant="link" className="py-4 text-white {文字太さ普通} {下線(あったほうがよさそう)}" asChild>
          <Link
            href="https://aback-jasmine-06b.notion.site/cf184497a6414a97aea49c1f1b2f5b5f"
          >
            プライバシーポリシー
          </Link>
        </Button>
```